README.md

```diff
@@ -98,23 +38,15 @@ After you're finished please remove all the comments and instructions!
 
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
+### 🌸 Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
```

Comment:
[OPTIONAL]
- [ ] Please kindly remove these information as they only to guide you in customizing this section :+1: :sparkles:

Reply:
Hello @Efiamotu-1 
I have already modified this part. 
Thanks for the suggesstion,
Best regards,